### DISCUSSION TITLE
Needed Refresh after mobile device go to "suspend" mode

### DISCUSSION THREAD

**Discussion by Emilot** [jcorporation/myMPD#1240] [Q&A]
myMPD version: [latest stable]

**Describe the bug**

When my phone(androidi or ios) or the iPad goes to lock screen ("suspend") mode and then i enable it again, i need to refresh the page, because i get the message "myMPD is disconnected". Can this happen automatically and show me at the state i was on?

Thanx!

**Comment by jcorporation** [comment]
Smartphones are closing the websocket connection if the app is in the background or suspended. Unfortunately the websocket state is not communicated correctly to the webapp. myMPD monitors the websocket through a ping/pong. The app should reconnect after a few seconds.

**Comment by Emilot** [comment]
Thank you JC!
